firestore-counter/PREINSTALL.md

```diff
@@ -18,7 +18,7 @@ Before installing this extension, make sure that you've [set up a Cloud Firestor
 After installing this extension, you'll need to:
 
 - Update your [database security rules](https://firebase.google.com/docs/rules).
-- Set up a [scheduled function](https://firebase.google.com/docs/functions/schedule-functions) to regularly call the controller function, which is created by this extension and monitors the extension's workload.
+- Set up a [Cloud Scheduler job](https://firebase.google.com/docs/functions/schedule-functions) to regularly call the controllerCore function, which is created by this extension and monitors the extension's workload.
```

Comment:
I'd like to update this docs hyperlink to: https://cloud.google.com/scheduler/docs/quickstart  WDYT?
(we'd do it throughout)

There's really no content on the Firesite page about Cloud Scheduler jobs.

Reply:
SGTM

Reply:
SGTM

Reply:
updated.  thx!